.github/workflows/scripts/release-against-charts.sh

```diff
@@ -0,0 +1,110 @@
+#!/bin/sh
+#
+# Bumps Webhook version in a locally checked out rancher/charts repository
+#
+# Usage:
+#   ./release-against-charts.sh <path to charts repo> <prev webhook release> <new webhook release>
+#
+# Example:
+# ./release-against-charts.sh "${GITHUB_WORKSPACE}" "v0.5.0-rc.13" "v0.5.0-rc.14"
+
+CHARTS_DIR=$1
+PREV_WEBHOOK_VERSION=$2   # e.g. v0.5.2-rc.3
+NEW_WEBHOOK_VERSION=$3    # e.g. v0.5.2-rc.4
+
+usage() {
+    cat <<EOF
+Usage:
+  $0 <path to charts repo> <prev webhook release> <new webhook release>
+EOF
+}
+
+bump_patch() {
+    version=$1
+    major=$(echo "$version" | cut -d. -f1)
+    minor=$(echo "$version" | cut -d. -f2)
+    patch=$(echo "$version" | cut -d. -f3)
+    new_patch=$((patch + 1))
+    echo "${major}.${minor}.${new_patch}"
```

Comment:
If perl is available, this is a one-liner:
```
echo $1 | perl -pe 's/(.*)(\d+)$/$1 . ($2 + 1)/e'
```

Reply:
Yeah it's also possible to do a one liner with awk, eg:

```sh
echo "v1.2.3" | awk -F. '{$3=$3+1;print}' OFS=.
```

but I opted to go with simple commands that I think more people are familiar with (eg: `cut` and addition).

Reply:
Disagreements accepted. Because of its heritage Perl is ubiquitous, but the only reason I would ever use it now is for its substitute-with-eval feature. I also learned last week that Perl is still on version 5, which shipped about 30 years ago, and probably always will be.